cli/pom.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jboss.pnc.build.finder</groupId>
+    <artifactId>build-finder-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
I am not sure as the coordinates are different anyway. It has the old tags but these are also using the koji-build-finder name.  The next logical version would have been 1.6.0, but given all the restructuring either 1.0.0 or 2.0.0 seems best.

Reply:
The 2.0.0 sounds better to me as well.

Reply:
Note that the location of the config file is also changing from `$HOME/.koji-build-finder` to `$HOME/.build-finder`. It sort of has to be like this as the old Infinispan file caches won't work since the class names/locations have changed.

Reply:
OK